styles/variables.scss

```diff
@@ -10,29 +10,44 @@ $blue: #4250e4;
 $blue-dark: #3b49e0;
 $purple: #9669ed;
 $success: #35e0ad;
-$red: #eb5757;
+$danger: #eb5757;
 $secondary: $purple;
 $warning: #fd8b2a;
 $orange: #EE9240;
-
-$blue-info: #87C7EC;
+$info: #87C7EC;
 
 $bg: #0d0f19;
+$bg-opac: rgba($white, 0.1);
 $bg-shadow: #151720;
 $bg-blue-shadow: #1825AD;
-
 $toast-font-size: 1rem;
 $m: 16px;
 
-$more-bg: ('white-50': $gray, 'orange': $warning);
+$more-bg: (
+        'white-50': $gray,
+        'shadow': $bg-shadow,
+        'opac': $bg-opac,
+        'purple': $purple,
+        'blue': $blue,
+        'orange': $orange, 
+        'success': $success
+);
+
+$more-colors: (
+        'gray': $gray,
+        'orange': $orange,
+        'purple': $purple,
+        'success': $success,
```

Comment:
this one doesn't need to be added here, instead it should be overriden on $theme-colors variable